lib/ansible/modules/cloud/vmware/vmware_vsan_health_facts.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/python
+
+# (c) 2019, OVH SAS
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: vmware_vsan_health_facts
+
+short_description: Gather facts about a VMware vSAN cluster's health.
+
+version_added: "2.9"
+
+description:
+    - "Gather facts about a VMware vSAN cluster's health"
+
+options:
+    cluster_name:
+        description:
+            - Name of the vSAN cluster
+        required: true
+        type: string
+    fetch_from_cache:
+        description:
+            - True to return the result from cache directly instead of running the full health check.
+        required: false
+        default: false
```

Comment:
No default value is given in the code, or is it something automatically set to `False` ?

`argument_spec.update(dict(fetch_from_cache=dict(required=False, type='bool')))`

Reply:
Yes it's False by default in the API